packages/jest-environment-puppeteer/src/readConfig.js

```diff
@@ -1,10 +1,7 @@
-import fs from 'fs'
-import path from 'path'
-import { promisify } from 'util'
-import cwd from 'cwd'
 import merge from 'merge-deep'
+import { loadPackageJSONFile, loadCustomConfig } from './util'
 
-const exists = promisify(fs.exists)
+const PACKAGE_JSON_KEY = 'jestPuppeteerConfig'
```

Comment:
Since the Jest config in `package.json` is just `jest`, maybe this could instead be `jestPuppeteer`? Or lump it under `jest` to a `puppeteer` property?